src/Metadata/Manifest/DefaultManifestSerializer.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\DbExtractor\Metadata\Manifest;
+
+use Keboola\Datatype\Definition\GenericStorage;
+use Keboola\DbExtractor\Metadata\Manifest\ManifestSerializer;
+use Keboola\DbExtractor\Metadata\ValueObject\Column;
+use Keboola\DbExtractor\Metadata\ValueObject\Table;
+
+class DefaultManifestSerializer implements ManifestSerializer
```

Comment:
Rovnako som vytvoril `ManifestSerializer` pre table/column metadata do manifestu, ... tento kod funguje rovnako ako povodny, ... prechadzaju aj povodne testy .... ak by ste vedeli o nejakom zozname metadat co KBC ocakava, tak to mozem este skontrolovat.